src/ontology/imports/merged_import.owl

```diff
@@ -347658,21 +348715,17 @@ Classes for population already  exist in IDO (&apos;organism population&apos;, I
     <!-- http://purl.obolibrary.org/obo/UBERON_0003942 -->
 
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/UBERON_0003942">
-        <owl:equivalentClass>
-            <owl:Class>
-                <owl:intersectionOf rdf:parseType="Collection">
-                    <rdf:Description rdf:about="http://purl.obolibrary.org/obo/UBERON_0001032"/>
-                    <owl:Restriction>
-                        <owl:onProperty rdf:resource="http://purl.obolibrary.org/obo/BFO_0000050"/>
-                        <owl:someValuesFrom rdf:resource="http://purl.obolibrary.org/obo/UBERON_0000012"/>
-                    </owl:Restriction>
-                </owl:intersectionOf>
-            </owl:Class>
-        </owl:equivalentClass>
```

Comment:
@shawntanzk do you see what I mean?

Reply:
yup this removal should solve it :) thanks!